test/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/eventException/ee001.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package com.ibm.jvmti.tests.eventException;
+
+import java.lang.reflect.Method;
+
+public class ee001 
+{
+    public native static void invoke(Method m);
+    public native static boolean check();
+    
+    public String helpException()
+    {
+        return "Test only single exception event gets thrown with JNI frame before handler";
+    }
+    
+    public static void generateException() throws Exception
+    {
+        throw new Exception();
+    }
+
+    public boolean testException()
+    {
+        boolean exceptionCaught = false;
+        try {
+            Method m = ee001.class.getMethod("generateException");
```

Comment:
This shouldn't be in the same try/catch as the `invoke()` call as a reflection lookup failure would make the test appear to pass.  Better to move it out and have the test method `throw Throwable`

Reply:
fixed